### DISCUSSION TITLE
Publishing GitHub Action to Marketplace gives 500 internal server error

### DISCUSSION THREAD

**Discussion by skosijer** [community/community#113415] [Actions]
### Select Topic Area

Bug

### Body

Publishing the GitHub Action from my organization repository gives 500 internal server error from GitHub.

I am trying to publish my GitHub Action to the Marketplace.

When I publish from my personal GitHub account repository, there are no issues and I'm able to tag the latest commit and create a release with the option to also publish to GitHub Marketplace. It gets published successfully.
This is the repository I used: https://github.com/skosijer/gh-action

When I try to publish from my organization GitHub repository, with exactly the same code and same process, **it gives me 500 internal server error**.
Here is the repository that causes the error: https://github.com/aspect-dev/gh-action

**TLDR;**
Publishing GitHub Action from my personal GitHub account repository works without any issues.
Publishing GitHub Action from my organization GitHub account repository gives 500 internal server error.

### Reproduction Steps

- Using this repository https://github.com/aspect-dev/gh-action try publishing a release from tag v1 and turn on the `Publish this Action to the GitHub Marketplace` option.
- The POST to https://github.com/aspect-dev/gh-action/releases returns 500 internal server error with standard GitHub error 500 screen.

**Comment by awcodify** [comment]
same here https://github.com/awcodify/comment-finder

**Reply by ericksonlbs** [reply]
same here https://github.com/saeg/jaguarportal-submit/

**Comment by awcodify** [comment]
The issue is resolved for me. It occurred because I had previously published an action with the same name, then deleted both the action and the repository. When I created a new repository and tried to publish an action with the same name, I encountered a 500 error. After create support ticket and the GitHub team cleaned up the previous data, I was able to publish the action successfully.

**Reply by ericksonlbs** [reply]
It's the same case. But I don't pay GitHub, so I can't open the ticket. I will open new discussion.

**Reply by kogut** [reply]
> It's the same case. But I don't pay GitHub, so I can't open the ticket. I will open new discussion.

Were you able to resolve it without paying?  I have the same issue.
